Tools/jit/_targets.py

```diff
@@ -120,6 +123,7 @@ async def _compile(
     ) -> _stencils.StencilGroup:
         o = tempdir / f"{opname}.o"
         args = [
+            *shlex.split(self.cflags),
```

Comment:
You know what? I changed my mind.

Let's move these to the end, so people can override our options if they really need to. Anyone using this option should know what they're doing.